x-pack/legacy/plugins/siem/public/components/page/hosts/host_overview/index.tsx

```diff
@@ -165,7 +170,7 @@ export const HostOverview = React.memo<HostSummaryProps>(
     ];
 
     return (
-      <OverviewWrapper>
+      <OverviewWrapper onMouseEnter={() => setIsHover(true)} onMouseLeave={() => setIsHover(false)}>
```

Comment:
Just out of curiosity, is there a simpler way to apply the CSS hover styles? It seems a little clunky to have to set the state of the hover rather than just using pure CSS `:hover` pseudo class.

Reply:
maybe the isHover variable is not the right name, but this triggering the prop `show`. I do not think that you can do that in pure CSS.